daft/scarf_telemetry.py

```diff
@@ -0,0 +1,92 @@
+import platform
+import urllib.parse
+import urllib.request
+from typing import Union
+
+from daft import get_build_type, get_version
+
+
+def scarf_telemetry(scarf_opt_out: bool, runner: str) -> tuple[Union[str, None], Union[str, None]]:
```

Comment:
since we don't care about the return, we can just drop that
```suggestion
def scarf_telemetry(scarf_opt_out: bool, runner: str)
```